services/brig/src/Brig/API/User.hs

```diff
@@ -607,32 +607,42 @@ changeSelfEmail u email allowScim = do
 -- | Prepare changing the email (checking a number of invariants).
 changeEmail :: UserId -> Email -> AllowSCIMUpdates -> ExceptT ChangeEmailError (AppT r) ChangeEmailResult
 changeEmail u email allowScim = do
-  em <-
-    either
-      (throwE . InvalidNewEmail email)
-      pure
-      (validateEmail email)
-  let ek = userEmailKey em
-  blacklisted <- lift . wrapClient $ Blacklist.exists ek
+  eml <- either (throwE . InvalidNewEmail email) pure (validateEmail email)
+  let eky = userEmailKey eml
```

Comment:
I personally find the name `eky` weird. But ok :-)

I would use `eKey`, `emailKey` or stick with `ek`